test-status/setup-db.js

```diff
@@ -47,6 +48,15 @@ function setupDb(db) {
     table.foreign('headSha')
         .references('pullRequestSnapshots.headSha')
         .onDelete('CASCADE');
+  }).createTable('buildCop', table => {
+    table.comment(
+        'Singleton table to store the GitHub username of active build cop');
```

Comment:
Can we ensure that this is true at the db level? Maybe by adding constraints or primary keys to the table.

Reply:
Not sure how. It's a simple enough app that I don't worry about this breaking (and if it does, recovery is easy enough) but if you know of a way I'll be happy to implement it!

Reply:
Sure maybe overkill for now :)